packages/gestalt/src/TextField.js

```diff
@@ -23,7 +23,7 @@ type Props = {
   /**
    * Indicate if autocomplete should be available on the input, and the type of autocomplete. Autocomplete values are implemented upon request. [Reach out to the Gestalt team](https://gestalt.pinterest.systems/team_support/get_help#Slack-channels) if you need [additional autocomplete values](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#values) to be supported.
    */
-  autoComplete?: 'bday' | 'current-password' | 'email' | 'new-password' | 'on' | 'off' | 'username',
+  autoComplete?: autoCompleteType,
```

Comment:
I think this is fine. Importing on this file.

Reply:
Yes, so it only ever has to be updated in one place, and it's ok since the docs are overriden.